innkeeper/eskiptojson.go

```diff
@@ -5,6 +5,59 @@ import (
 	"log"
 )
 
+type matchType string
```

Comment:
true.. i just didn't want to change the write direction this time. i'll do that in a next round. this one is only needed for the other, unused, code to compile.